features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql

```diff
@@ -2812,6 +2812,53 @@ CREATE TRIGGER IDN_OAUTH2_AUTHZ_CODE_ACTOR_TRIG NO CASCADE
         END
 /
 
+CREATE TABLE VC_CONFIG (
+    ID CHAR(36) NOT NULL,
+    CURSOR_KEY INTEGER NOT NULL,
+    TENANT_ID INTEGER NOT NULL,
+    IDENTIFIER VARCHAR(255) NOT NULL,
+    DISPLAY_NAME VARCHAR(255) NOT NULL,
```

Comment:
Shall we introduce a description column also?

Reply:
fixed